src/components/Book.js

```diff
@@ -1,20 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 function Book({ book, removeBook }) {
   return (
     <div>
       <div className="book-section" key={book.item_id}>
         <div className="book-innerSection">
           <span className="book-category">{book.category}</span>
-          <p className="book-title">{book.title}</p>
+          <h5 className="book-title">{book.title}</h5>
           <span className="book-author">{book.author}</span>
           <div className="btn-container">
```

Comment:
- Great Job for implementing previous reviiwer seggetion, I think you missed one thing the `book` and `category`  menus
so remove the `a` default style and fit the given design as the previous reviewer suggested to have a better design 👍 
<img width="534" alt="2023-11-04_13-41" src="https://github.com/Obimbo07/book-library/assets/52010789/9176dd7a-7358-44fc-a6df-1d8005fdd4b5">